frontend/packages/process-editor/src/components/ConfigPanel/ConfigEndEvent/CustomReceiptContent/CustomReceipt/CustomReceipt.test.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import { CustomReceipt } from './CustomReceipt';
+import { render, screen } from '@testing-library/react';
+import { textMock } from '../../../../../../../../testing/mocks/i18nMock';
+import { BpmnContext } from '../../../../../contexts/BpmnContext';
+import userEvent from '@testing-library/user-event';
+import { BpmnApiContext, type BpmnApiContextProps } from '../../../../../contexts/BpmnApiContext';
+import { BpmnConfigPanelFormContextProvider } from '../../../../../contexts/BpmnConfigPanelContext';
+import {
+  mockBpmnApiContextValue,
+  mockBpmnContextValue,
+} from '../../../../../../test/mocks/bpmnContextMock';
+import { type LayoutSetConfig } from 'app-shared/types/api/LayoutSetsResponse';
+import { PROTECTED_TASK_NAME_CUSTOM_RECEIPT } from 'app-shared/constants';
+
+const invalidFormatLayoutSetName: string = 'Receipt/';
+const emptyLayoutSetName: string = '';
+const existingLayoutSetName: string = 'layoutSetName1';
+
+const existingCustomReceiptLayoutSetId: string = mockBpmnApiContextValue.layoutSets.sets[0].id;
+const layoutSetWithCustomReceipt: LayoutSetConfig = {
+  id: existingCustomReceiptLayoutSetId,
+  tasks: [PROTECTED_TASK_NAME_CUSTOM_RECEIPT],
+};
+const layoutSetWithDataTask: LayoutSetConfig = {
+  id: existingLayoutSetName,
+  tasks: ['Task_1'],
+};
+
+const layoutSetIdTextKeys: Record<string, string> = {
+  [emptyLayoutSetName]: 'validation_errors.required',
+  [invalidFormatLayoutSetName]: 'ux_editor.pages_error_format',
+  [existingLayoutSetName]: 'process_editor.configuration_panel_layout_set_id_not_unique',
+  [existingCustomReceiptLayoutSetId]: '', // Define the text key for this layoutSetId
+};
```

Comment:
Does the last one need to be in the object? If not, maybe a better name could be layoutSetIdErrorTextKeys